iop/ci/complete-iop-values.yaml

```diff
@@ -80,5 +81,29 @@ storageprovider-reva:
       [http.services.dataprovider]
       driver = "local"
 
+storageprovider-public:
+  enabled: true
+  service:
+    grpc:
+      port: 16000
+    http:
+      port: 16001
+  configFiles:
+    revad.toml: |
```

Comment:
@dagl I'm not sure this file matches the contents of [the `storage-public.toml` example](https://github.com/cs3org/reva/blob/master/examples/storage-references/storage-public.toml)

Reply:
Sorry, overlook that one

Reply:
Ok, I'll fix that in a separate PR